standalone/src/main/java/io/delta/standalone/types/StructType.java

```diff
@@ -108,6 +110,11 @@ public String getTreeString() {
         return builder.toString();
     }
 
+    public MessageType asParquet() {
```

Comment:
Do we need this interface here? It's better to add a new class similar to Spark's ParquetToSparkSchemaConverter.

Reply:
My thinking was that this method would return the result of internally using `ParquetToSparkSchemaConverter`.

So we would essentially call `myParquetToSparkSchemaConverter.convert(this)`

```
/**
   * Converts a Spark SQL [[StructType]] to a Parquet [[MessageType]].
   */
  def convert(catalystSchema: StructType): MessageType = {
```

Reply:
So I was thinking we keep that class internal and just expose the structType -> parquetType conversion functionality

Reply:
This is to avoid the basic `StructType` depending on Parquet APIs.

Reply:
sounds good